packages/create-subscription/src/createSubscription.js

```diff
@@ -126,17 +120,16 @@ export function createSubscription<Property, Value>(
         };
 
         // Store subscription for later (in case it's needed to unsubscribe).
-        // This is safe to do via mutation since:
-        // 1) It does not impact render.
-        // 2) This method will only be called during the "commit" phase.
         const unsubscribe = subscribe(source, callback);
-
         invariant(
           typeof unsubscribe === 'function',
           'A subscription must return an unsubscribe function.',
         );
 
-        this.state.unsubscribeContainer.unsubscribe = unsubscribe;
+        // Storing unsubscribe is safe to do via mutation since:
+        // 1) It does not impact render.
+        // 2) This method will only be called during the "commit" phase.
+        this.state.unsubscribe = unsubscribe;
```

Comment:
Why not a field?

Reply:
A "field"?

Reply:
If you're asking, "Why store in state rather than on the instance?"

I guess I _could_ do that, so long as I always unsubscribed _before_ subscribing in `componentDidUpdate` 🤔

Reply:
I guess it "felt" a little nicer to group the source, value, and unsubscribe all together within a single `state` object- but it's subjective.

Reply:
To me, mutating state feels stranger than splitting them up. e.g. we might freeze the state object in the future.

Reply:
Makes sense!

Are you happier with what I changed it to?

Reply:
Looks good to me! I trust you to know when it's safe to mutate things 😛